src/clr-angular/layout/tabs/tab-link.directive.ts

```diff
@@ -85,7 +85,7 @@ export class ClrTabLink {
 
   @HostBinding('attr.role')
   get role(): string {
-    return 'presentation';
+    return 'tab';
```

Comment:
Note: We have a [guideline](https://github.com/vmware/clarity/blob/master/CODING_GUIDELINES.md#code-style) to put host bindings that are *not* dynamic as part of the `host` config in the component metadata, and use `@HostBinding` for the ones that are dynamic.

This component is doing the exact opposite. You can either fix this now, or we make a note of it somewhere either in comments or as a new issue.